### ISSUE TITLE
warnings when running the inference and no music

### ISSUE THREAD

**Issue by sammyf** [deepbeepmeep/YuEGP#11]
specs : archlinux, Python 3.10.11, RTX3090 + A1000 
when starting gradio I get 
`
You are attempting to use Flash Attention 2.0 with a model not initialized on GPU. Make sure to move the model to GPU after initializing it on CPU with `model.to('cuda')`.`

When I try to generate anything I always get this  warning : 

```
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
/media/GINTONIC/AIArtists/Music-Generators/YuEGP/yuenv/lib/python3.10/site-packages/transformers/generation/utils.py:2137: UserWarning: You are calling .generate() with the `input_ids` being on a device type different than your model's device. `input_ids` is on cuda, whereas the model is on cpu. You may experience unexpected behaviors or slower generation. Please make sure that you have put `input_ids` to the correct device by calling for example input_ids = input_ids.to('cpu') before running `.generate()`.
  warnings.warn(
```

The resulting track has only vocals, but no music. 

I tried hardcoding 'cuda:0' as device, in case the two GPUs are the problem, but that didn't help. I also tried to force the input_ids to cpu before the calls to .generate, didn't help neither.

these are the installed python packages : 

```
Package                  Version
------------------------ ------------
absl-py                  2.1.0
accelerate               1.3.0
aiofiles                 23.2.1
annotated-types          0.7.0
antlr4-python3-runtime   4.9.3
anyio                    4.8.0
argbind                  0.3.9
asttokens                3.0.0
audioread                3.0.1
certifi                  2025.1.31
cffi                     1.17.1
charset-normalizer       3.4.1
click                    8.1.8
contourpy                1.3.1
cycler                   0.12.1
decorator                5.1.1
descript-audio-codec     1.0.0
descript-audiotools      0.7.2
docstring_parser         0.16
einops                   0.8.0
exceptiongroup           1.2.2
executing                2.2.0
fastapi                  0.115.8
ffmpy                    0.5.0
filelock                 3.13.1
fire                     0.7.0
flash_attn               2.7.4.post1
flatten-dict             0.4.2
fonttools                4.55.8
fsspec                   2024.6.1
future                   1.0.0
gradio                   5.14.0
gradio_client            1.7.0
grpcio                   1.70.0
h11                      0.14.0
httpcore                 1.0.7
httpx                    0.28.1
huggingface-hub          0.28.1
idna                     3.10
importlib_resources      6.5.2
ipython                  8.32.0
jedi                     0.19.2
Jinja2                   3.1.4
joblib                   1.4.2
julius                   0.2.7
kiwisolver               1.4.8
lazy_loader              0.4
librosa                  0.10.2.post1
llvmlite                 0.44.0
Markdown                 3.7
markdown-it-py           3.0.0
markdown2                2.5.3
MarkupSafe               2.1.5
matplotlib               3.10.0
matplotlib-inline        0.1.7
mdurl                    0.1.2
mmgp                     3.1.4.post15
mpmath                   1.3.0
msgpack                  1.1.0
networkx                 3.3
ninja                    1.11.1.3
numba                    0.61.0
numpy                    1.26.4
nvidia-cublas-cu12       12.4.5.8
nvidia-cuda-cupti-cu12   12.4.127
nvidia-cuda-nvrtc-cu12   12.4.127
nvidia-cuda-runtime-cu12 12.4.127
nvidia-cudnn-cu12        9.1.0.70
nvidia-cufft-cu12        11.2.1.3
nvidia-curand-cu12       10.3.5.147
nvidia-cusolver-cu12     11.6.1.9
nvidia-cusparse-cu12     12.3.1.170
nvidia-nccl-cu12         2.21.5
nvidia-nvjitlink-cu12    12.4.127
nvidia-nvtx-cu12         12.4.127
omegaconf                2.3.0
optimum-quanto           0.2.6
orjson                   3.10.15
packaging                24.2
pandas                   2.2.3
parso                    0.8.4
peft                     0.14.0
pexpect                  4.9.0
pillow                   11.0.0
pip                      25.0
platformdirs             4.3.6
pooch                    1.8.2
prompt_toolkit           3.0.50
protobuf                 3.19.6
psutil                   6.1.1
ptyprocess               0.7.0
pure_eval                0.2.3
pycparser                2.22
pydantic                 2.10.6
pydantic_core            2.27.2
pydub                    0.25.1
Pygments                 2.19.1
pyloudnorm               0.1.1
pyparsing                3.2.1
pystoi                   0.4.1
python-dateutil          2.9.0.post0
python-multipart         0.0.20
pytz                     2025.1
PyYAML                   6.0.2
randomname               0.2.1
regex                    2024.11.6
requests                 2.32.3
rich                     13.9.4
ruff                     0.9.4
safehttpx                0.1.6
safetensors              0.5.2
scikit-learn             1.6.1
scipy                    1.10.1
semantic-version         2.10.0
sentencepiece            0.2.0
setuptools               65.5.0
shellingham              1.5.4
six                      1.17.0
sniffio                  1.3.1
soundfile                0.13.1
soxr                     0.5.0.post1
stack-data               0.6.3
starlette                0.45.3
sympy                    1.13.1
tensorboard              2.18.0
tensorboard-data-server  0.7.2
termcolor                2.5.0
threadpoolctl            3.5.0
tokenizers               0.21.0
tomlkit                  0.13.2
torch                    2.5.1+cu124
torch-stoi               0.2.3
torchaudio               2.5.1+cu124
torchvision              0.20.1+cu124
tqdm                     4.67.1
traitlets                5.14.3
transformers             4.48.1
triton                   3.1.0
typer                    0.15.1
typing_extensions        4.12.2
tzdata                   2025.1
urllib3                  2.3.0
uvicorn                  0.34.0
wcwidth                  0.2.13
websockets               14.2
Werkzeug                 3.1.3
wheel                    0.45.1

```

**Comment by deepbeepmeep** [comment]
the warnings comes from the transformers library and should be ignored. Sometime one track is missing and it seems that using a different seed or increasing the number of segments to generate fix that.
